src/app/static/src/js/report.js

```diff
@@ -22,4 +23,12 @@ export const vm = new Vue({
             this.report.published = !this.report.published
         }
     }
+});
+
+export const runVm  = new Vue({
+    el: '#runReportVueApp',
```

Comment:
Just realised that we should be checking that elements exist before attaching vue apps, since they won't always be on the page and this will cause a js error - applies to the publish switch as well!